autoware_launch/launch/autoware.launch.xml

```diff
@@ -4,10 +4,10 @@
   <arg name="map_path" description="point cloud and lanelet2 map directory path"/>
   <arg name="vehicle_model" description="vehicle model name"/>
   <arg name="sensor_model" description="sensor model name"/>
-  <arg name="use_pointcloud_container" default="true" description="launch pointcloud container"/>
   <arg name="pointcloud_container_name" default="pointcloud_container"/>
```

Comment:
Should `pointcloud_container_name` be optional as well? :thinking:

Reply:
ops...
Yes, you are correct.

Reply:
https://github.com/tier4/autoware_launch/pull/197/commits/6165a30180b96e9f9b015f8a9a9b6fe67ad318e7